sapl/templates/norma/normajuridica_form.html

```diff
@@ -10,20 +10,28 @@
     var tipo_materia = $("#id_tipo_materia").val();
     var numero_materia = $("#id_numero_materia").val();
     var ano_materia = $("#id_ano_materia").val();
-    var tipo = $('#id_tipo').val();
-    var ano = $('#id_ano').val();
-    var numero = $('#id_numero').val();
-    var ementa = $('#id_ementa').val();
+
     if (tipo_materia && numero_materia && ano_materia) {
-      $.get("/sessao/recuperar-materia",
-            {tipo_materia: tipo_materia,
-             numero_materia: numero_materia,
-             ano_materia: ano_materia},
-        function(data, status) {
-          $("#id_ementa").val(data.ementa);
-          $("#id_indexacao").val(data.indexacao);
-        }
-      );
+      $.get("/sessao/recuperar-materia", {
+              tipo_materia: tipo_materia,
+              numero_materia: numero_materia,
+              ano_materia: ano_materia
+          }, (data, status) => {
+            if ($(location).attr('pathname').split("/")[2] == "create") {
```

Comment:
Wow, muito cuidado aqui! Ficar manipulando path assim deixa o código muito frágil, pois qualquer mudança no path e a tela deixa de funcionar. Melhor seria evitar isso aqui. Por sinal, qual a razão deste if? Se é somente pra mostrar a modal não seria o caso de fazer o binding no blur (ou no focus out, o que funcionar melhor) de ambos os campos (número e ano)?

Reply:
Há a verificação para mostrar a modal somente quando for a criação do objeto. Não tem sentido notificar a mudança de algo se não há mudança.

Reply:
> Não tem sentido notificar a mudança de algo se não há mudança

A questão não é essa. Faça de uma forma que **não envolva** código como da linha 20.

Reply:
Sim, esse _form_ é para criação ou atualização da norma. Alguma sugestão de como identificar pelo jquery qual ação o _form_ tá executando?